_api-reference/index-apis/clear-index-cache.md

```diff
@@ -18,7 +18,7 @@ If you use the Security plugin, you must have the `manage index` privileges.
 | Parameter | Data type | Description |
 :--- | :--- | :---
 | target | String | Comma-delimited list of data streams, indexes, and index aliases to which cache clearing will be applied. Wildcard expressions (`*`) are supported. To target all data streams and indexes in a cluster, omit this parameter or use `_all` or `*`. Optional. |
-
+| file | Boolean | If `true`, clears the unused entries from the filecache on 'search' capable nodes. Defaults to `false`. |
```

Comment:
Should it be "file cache" (two words)? As written, it should technically be "`search`-capable nodes" (as in the heading below), but we should avoid that for formatting reasons. Can we rephrase?

Reply:
Move line 21 to the Query parameter table. This is not a path parameter.